requirements.txt

```diff
@@ -1,6 +1,7 @@
 adal==0.4.3
 applicationinsights==0.10.0
 argcomplete==1.8.0
+azure_devtools==0.1.0
```

Comment:
And I don't think it needs to be listed in requirements list. In dependencies is good enough.